src/AirspaceConverter.h

```diff
@@ -33,6 +33,7 @@ class AirspaceConverter {
 		SeeYou_Format,
 		Polish_Format,
 		Garmin_Format,
+		CSV_Format,
```

Comment:
Forse dovremmo differenziare tra formati di spazi aerei e wayspints...

Reply:
in effetti SeeYou e CSV sono punti o trail e non airspaces, Polish e Garmin non li conosco